src/graphene-matrix.c

```diff
@@ -746,7 +746,25 @@ graphene_matrix_get_value (const graphene_matrix_t *m,
       return 0.f;
     }
 
-  return graphene_simd4f_get (r, col);
+  switch (col)
```

Comment:
Ah, that would have been clearer in a separate commit indeed. ;-)

Yes, if you could split this change in a separate commit and note why it's necessary in the commit log it would be very much welcome — especially because it'll avoid me getting rid of it in the future when I've forgotten about this issue.